pkg/datasource/apollo/go.mod

```diff
@@ -0,0 +1,51 @@
+module github.com/alibaba/sentinel-golang/pkg/datasource/apollo
+
+go 1.17
```

Comment:
1.17 might be too new for some legacy users. How about keeping 1.13 (if possible?)

Reply:
OK